cloudscript/cloudscript/CreateImage.ps1

```diff
@@ -0,0 +1,93 @@
+﻿[CmdletBinding()]
+Param(
+    [Parameter(Mandatory=$True)]
+    [ValidateNotNullOrEmpty()]
+    # Image name to be created
+    [string]$imageName,
+
+    [Parameter(Mandatory=$True)]
+    [ValidateNotNullOrEmpty()]
+    # Image resource group for above image
+    [string]$imageResourceGroup,
+
+    [Parameter(Mandatory=$True)]
+    [ValidateNotNullOrEmpty()]
+    # Blob sas uri from which image to be created
+    [string]$blobUri,
+
+    [Parameter(Mandatory=$True)]
+    [ValidateSet('Windows','Linux')]
+    # os type for image: Linux/Windows
+    [string]$osType,
+
+    [Parameter(Mandatory=$True)]
+    [ValidateNotNullOrEmpty()]
+    # Resource group containing azure device
+    [string]$saasResourceGroup,
+
+    [Parameter(Mandatory=$True)]
+    [ValidateNotNullOrEmpty()]
+    # Azure ASE device name
+    [string]$deviceName,
+
+    [Parameter(Mandatory=$True)]
+    [ValidateNotNullOrEmpty()]
+    # Azure subscription id
+    [string]$subscriptionId,
+
+    [Parameter(Mandatory=$True)]
+    [ValidateNotNullOrEmpty()]
+    # Access key for azure subscription access
+    [string]$saasAccessKey,
+
+    [Parameter(Mandatory=$True)]
```

Comment:
Can we have a link to the following article here which talks about how to obtain the tenant ID and the client ID?
https://docs.microsoft.com/en-us/azure/databox-online/azure-stack-edge-gpu-deploy-virtual-machine-cli-python